### PR TITLE
Add loading and error from evergreen to DataWrapper

### LINKED ISSUE #50: Create atom for loading and error
### Description
We need custom atoms for loading and errors which can be displayed when fetching data from the backend. 

### Subtasks
- [x] Create atom for loading as designed in #32 
- [x] Create atom for error as designed in #32
- [x] Create atom for cases where GraphQL didn't fetch any data (as designed in #32)
- [x] Implement atoms in DataWrapper.tsx and Dashboard.tsx

### PR DESCRIPTION
**Pull request by emilom**
Closes #50 

DataWrapper will now handle loading and error messages from GraphQL. This will be displayed on the dashboard in DashboardItems.

### CODE REVIEW

**Review by blauks — COMMENTED** [review]
Hey Emil, this LGTM to me! But I think it would be even better if you added a Storybook item for each of the different scenarios!🙂

**Comment by emilom** [comment]
> Hey Emil, this LGTM to me! But I think it would be even better if you added a Storybook item for each of the different scenarios!🙂

Great idea! Thanks for the input, will take a look at storybook now 👊

File: src/components/molecules/DataWrapper.tsx [hunk 1]
```diff
@@ -11,23 +12,44 @@ export type DataWrapperProps<T, U> = {
...
 };
 
+const CenterPane: React.FC = ({ children }) => {
+    return (
+        <Pane width="100%" height="100%" display="flex" justifyContent="center" alignItems="center">
+            {children}
+        </Pane>
+    );
+};
+
```
Comment (FredrikAugust) [inline src/components/molecules/DataWrapper.tsx]: Perhaps move to atom if it's going to be reused other places?

File: src/components/molecules/DataWrapper.tsx [hunk 2]
```diff
@@ -11,23 +12,44 @@ export type DataWrapperProps<T, U> = {
...
     }
 
     if (error) {
-        return <p>Error: {error.message}</p>;
+        return (
+            <CenterPane>
+                <InlineAlert intent="danger">Vi finner ikke dataen du leter etter.</InlineAlert>
+            </CenterPane>
+        );
     }
```
Comment (FredrikAugust) [inline src/components/molecules/DataWrapper.tsx]: Få med hva error-en er. Dette kan du finne i `error.message`.

Comment (FredrikAugust) [inline src/components/molecules/DataWrapper.tsx]: Få med hva error-en er. Dette kan du finne i `error.message`. 

You should already know this.

Comment (FredrikAugust) [reply src/components/molecules/DataWrapper.tsx]: Kanskje få med at et problem oppstod under henting av data. Typ "Det oppsto en feil under henting av data".

**Review by FredrikAugust — CHANGES_REQUESTED** [review]
Will the toasts be implemented in another PR? If so we should create a work package for that.

File: src/components/atoms/ErrorMessage.tsx [hunk 3]
```diff
@@ -0,0 +1,29 @@
+import { InlineAlert } from 'evergreen-ui';
+import React from 'react';
+import { Text, Pane } from 'evergreen-ui';
```
Comment (blauks) [inline src/components/atoms/ErrorMessage.tsx]:
```suggestion
import React from 'react';
import { Text, Pane, InlineAlert } from 'evergreen-ui';
```

File: src/components/atoms/ErrorMessage.tsx [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+export type ErrorMessageProps = {
+    size?: 300 | 400 | 500 | 600;
+    message?: string;
+    moreInfo?: string;
+};
+
+const ErrorMessage: React.FC<ErrorMessageProps> = ({ message = 'Error :(', moreInfo, size = 400 }) => {
```
Comment (FredrikAugust) [inline src/components/atoms/ErrorMessage.tsx:10]: A nifty trick is to replace the 

```
{
  return (
```

with just `(` (meaning implicit return).

You don't need to change this, but it is a helpful tool.

File: src/stories/ErrorMessage.stories.tsx [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+const sizeControl = { control: { type: 'radio', options: [300, 400, 500, 600] } };
+
+export const Primary = Template.bind({});
+Primary.args = { size: 400, message: 'Error message' };
+Primary.argTypes = {
+    size: sizeControl,
+};
+
+export const Secondary = Template.bind({});
+
```
Comment (blauks) [inline src/stories/ErrorMessage.stories.tsx:24]:
```suggestion
```

Comment (blauks) [reply src/stories/ErrorMessage.stories.tsx:24]: Just to keep everything nice and equal :)

**Comment by emilom** [comment]
> Will the toasts be implemented in another PR? If so we should create a work package for that.

No need for toast now, I'm not hungry 😋 
We might use it later.

**Review by FredrikAugust — DISMISSED** [review]

**Comment by blauks** [comment]
> > Will the toasts be implemented in another PR? If so we should create a work package for that.
> 
> No need for toast now, I'm not hungry 😋
> We might use it later.

Maybe for dinner 😁

**Review by FredrikAugust — APPROVED** [review]

**Pull request merged**